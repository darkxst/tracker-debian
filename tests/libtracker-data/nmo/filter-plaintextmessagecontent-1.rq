SELECT ?plainTextMessageContent
WHERE { ?x nmo:plainTextMessageContent ?plainTextMessageContent }
ORDER BY ?plainTextMessageContent
