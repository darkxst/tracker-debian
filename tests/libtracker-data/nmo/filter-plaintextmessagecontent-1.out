"stringly data for nmo:plainTextMessageContent"
